packages/app/src/cli/models/app/loader.ts

```diff
@@ -81,6 +87,8 @@ class AppLoader {
     const nodeDependencies = await getDependencies(packageJSONPath)
     const packageManager = await getPackageManager(this.appDirectory)
     const {webs, usedCustomLayout: usedCustomLayoutForWeb} = await this.loadWebs()
+    const pnpmWorkspacesPath = path.join(this.appDirectory, pnpmWorkspaceFile)
+    const usesWorkspaces = await appUsesWorkspaces(packageJSONPath, pnpmWorkspacesPath)
```

Comment:
What about passing just the app directory and generating `packageJSONPath` and `pnpmWorkspacesPath` inside? It would simplify a bit the usage and it would be consistent with other functions like `getPackageManager`.

Reply:
Good idea, I'll change it.